config/config.go

```diff
@@ -544,6 +543,7 @@ func Load(configFlag string, version string) *Config {
 	config.InstanceId = uuid.Must(uuid.NewV4()).String()
 	config.App.Colors = readColors()
 	config.Db.Dialect = resolveDbDialect(config.Db.Type)
+	slog.Info("loaded configuration", "environment", config.Env, "version", config.Version, "db_dialect", config.Db.Type)
```

Comment:
Why put this log line just exactly here, in the middle of config loading? Why not at the end?